mdk/build.gradle

```diff
@@ -4,7 +4,7 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath group: 'net.minecraftforge.gradle', name: 'ForgeGradle', version: '4.1.+', changing: true
+        classpath group: 'net.minecraftforge.gradle', name: 'ForgeGradle', version: '5.+', changing: true
```

Comment:
With how breaking changes get handled for FG, I feel like it may almost make sense to just hold off on this slightly longer until this can be 5.1.+ especially for purposes of the mdk to make it more clear about the .odd = stable, .even = breaking